### ISSUE TITLE
fix #748 problem

### ISSUE THREAD

**Issue by RavenB** [Athou/commafeed#750]
Hi,
apparently the fix hides all pictures in all feeds now and displays only the problemfeeds picture .
To fix the problem would be to hide the previeuw in the problem feed (the real double picture of http://rt.com/rss) (show all pictures everywhere (undo fix 748 but hide the currently shown rt.com picture which are in the feed twice.)

in short (hope this sums it up better) :  hide the preview (which seems to be only present in the rt feed) but display the enclosure. 7 other feeds (including bbc.com rss feeds) have no preview but do have an enclosure, so it would be better to hide the preview and not the enclosure  (display enclosure).

Basically the opposite of fix 748. concluding in do not display preview if enclosure present but do always display enclosure).

**Comment by Athou** [comment]
Can you give me a feed url where the preview image is not displayed because I can't reproduce what you experience.

**Comment by RavenB** [comment]
http://sputniknews.com/export/rss2/archive/index.xml (has only enclosure not a preview and no longer shows picture fix748 )
http://www.nasa.gov/rss/dyn/image_of_the_day.rss (no longer displays image since 748)

basically these feeds have enclosure but no preview.  (if we do not display preview if we have enclosure which we display always both issues would be solved). (i have a few additional rss feeds that also have no preview but enclosure only)

**Comment by Athou** [comment]
Sorry I didn't test properly, this one should fix it.

**Comment by RavenB** [comment]
Hi,
no prob can test as much you need. 
Just updated. Unfortunately back to how it used to be double pictures ( preview and enclosure are both displayed in the rt.com/rss feed) . Can you do "do not display preview if enclosure present and displayed" ?

**Comment by Athou** [comment]
I have a single image for the rt.com/rss feed, I'm not sure why you have two.

**Comment by RavenB** [comment]
Hi,
I forced a second git pull and rebuild and cache cleanup and can confirm its perfect now :) (sorry about that)
thank you for the fixes.!!

**Comment by Athou** [comment]
Indeed, good catch. Enclosures are now proxied too and it should fix the issue.
